specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/ManagedInstanceOperations.json

```diff
@@ -0,0 +1,358 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
```

Comment:
For future PRs please copy the previous version's swagger spec in commit 1 before making changes in future commits.  This makes it much easier to review.